### ISSUE TITLE
Implement quo2 - Settings - Category component

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#16211]
Implement the category component as per designs:

https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=9419-161493&t=btEL0EAzW3vIPPQ5-0

Please place the implementation in the quo2 preview.

Have a word/chat with our designers before starting to implement to clarify the interactions.
You should be re-using existing components where available.

**Comment by OmarBasem** [comment]
Opened a separate issue for reorder component: https://github.com/status-im/status-mobile/issues/16712
